docs/general/naming.md

```diff
@@ -309,4 +338,37 @@ freely. For example, `system.paging.faults` and `system.network.packets`.
 Units do not need to be specified in the names since they are included during
 instrument creation, but can be added if there is ambiguity.
 
+### Client and server metrics
+
+Metrics that measure aspects of a physical or logical network call should include
+an indication of the side being measured when ambiguity exists.
+
+In such cases, the metric name should follow the pattern: `{domain}.{client|server}.{metric_name}`.
+
+For example:
+- `http.client.request.duration`
+- `gen_ai.server.request.duration`
+- `messaging.client.sent.messages`
+- `messaging.process.duration` - the term `process` clearly indicates that
+  the metric is reported by the consumer.
+- `kestrel.connection.duration` - here, `kestrel` is the name of the web server,
+  so no additional indication is necessary.
+
+### System-specific metrics
+
+When a metric is specific to a system (e.g., project, provider, product) in a given domain,
+the system name should be included in the instrument name using the pattern:
+`{domain}.{client|server}.{system}.*.{property}` pattern.
+
+For example, `db.client.cosmosdb.operation.request_charge`
```

Comment:
the difference with `signalr.server.connection.duration` below makes me think if we should simplify it and do `cosmosdb.client.operation.request_charge`. What's the benefit of having `db` in front of it?

Reply:
summoning @open-telemetry/semconv-system-approvers 

It's the same question as for `system.linux` vs `linux` metrics in https://github.com/open-telemetry/semantic-conventions/issues/1403 or https://github.com/open-telemetry/semantic-conventions/pull/1618